althea_kernel_interface/src/exit_client_tunnel.rs

```diff
@@ -35,7 +34,7 @@ impl KernelInterface {
             ],
         )?;
 
-        let output = self.run_command(
+        let _output = self.run_command(
```

Comment:
if we dont use output I dont think we need to bind it to a variable at all

Reply:
I'm pretty sure _ and _output are the same the compiler will optimize out actually saving the value.